server/src/core/server/services/dsaReports/reports.ts

```diff
@@ -201,8 +205,22 @@ export async function makeDSAReportDecision(
       commentID,
       commentRevisionID,
       userID,
-      now
+      now,
+      undefined,
+      false
     );
+
+    if (comment.authorID) {
```

Comment:
We need to send out a notification to the reporter of the DSA report of the decision as well here, right?

Reply:
This is hooked up now, I just initially posted my work as a draft to keep visibility for this work available to y'all!